Modules/xASL_module_ASL.m

```diff
@@ -874,9 +874,4 @@
     end
 end
 
-
-%% 7. Define sequence (educated guess based on the Q field)
-x = xASL_adm_DefineASLSequence(x, bOutput);
-
-
-end
\ No newline at end of file
+end
```

Comment:
Also see my main comment, we need to either load x.Q.Sequence at 6 locations throughout the xASL_module_ASL or keep this. So probably best to keep this here (replace by the [~, ~, json] = xASL_io_ReadNifti ...) and remove at the end

Reply:
Don't worry. I have checked that to make sure it *is* backwards compatible.

In module_ASL at line 133, we call xASL_module_ASL_ParseParameters and this function makes sure that x-struct is loaded and parms.mat is loaded and everything from ASL4D.json is loaded. So I've only removed ASLDefineSequence because I knew that at the start of the ASL module, we will load the x.Q.Sequence from the JSON through parms2BIDS.

I agree that the whole stuff is convoluted etc. and we will get rid of _parms.mat in the future. But for now, we are remaining perfectly backwards compatible.

So this answers your questions and is in line with your suggestion, but we already have all that.

Reply:
I see, then we should only (now or in the future) make this more transparent, this what you're talking about is hidden in `xASL_module_ASL_ParseParameters` -> LoadParms -> BIDS2Parms -> DefineASLSequence.

Probably when we remove the parms.mat it is easier to simplify this. Should we move this to that issue? You can resolve this:)

Reply:
FUlly agree and added that to #1637